spec/components/site_setting_extension_spec.rb

```diff
@@ -140,6 +140,21 @@ def new_settings(provider)
         settings.set("test_setting", 12)
         expect(settings.test_setting).to eq(12)
       end
+
+      it "should publish changes to clients" do
+        settings.setting("test_setting", 100)
+        settings.client_setting("test_setting")
+        called = false
+
+        MessageBus.subscribe('/client_settings') do |msg|
+          expect(msg.data["value"]).to eq(88)
+          called = true
+        end
+
+        settings.test_setting = 88
+        sleep 0.01
```

Comment:
@SamSaffron Do you know if there is a way to monitor MessageBus so that I can wait until it publishes?